### ISSUE TITLE
SPN Module Blocks Streaming From PC, Even For Free Users

### ISSUE THREAD

**Issue by NikkMann** [safing/portmaster#1412]
Was trying to use Plex to stream tv shows/movies from my pc to my tv, and Nvidia Game Stream to stream games from my pc to my tablet.

I kept getting network errors in both whether I tried to do in-home streaming or internet streaming, and the way I solved this issue at first was by disabling the Use SPN option under those specific services. 

I then noticed that Use SPN was enabled for every service even though I had the SPN Module disabled globally, so I tried enabling the SPN Module setting so the sub-options appeared, found that Use SPN was on, turned it off, then turned SPN Module back off. This disabled the Use SPN option for all services.

I am a free user, so it shouldn't even be working if enabled, but for some reason disabling it fixes streaming issues.
In the future, having Use SPN also defaulted to off in portmaster instead of On when SPN Module is off could fix this issue.

I mentioned this in the Discord before and one of the admins recommended making a github issue about it.

On Windows 11, latest portmaster version.

**Comment by Raphty** [comment]
I recently did a video explaining how to allow incoming connections on YT
https://www.youtube.com/watch?v=NObJFzBaeW4

there is also this wiki page explaining how to do it
https://wiki.safing.io/en/FAQ/AllowIncomingConnections

this has nothing to do with SPN and no it was not running, the module does not nor do nodes accept a connection if you don't have permission to use the network.

**Comment by NikkMann** [comment]
I probably should have mentioned that all the Force Block settings were disabled globally and for the apps I was using, along with Windows connections.

That aside, this is one of the strangest responses I've had on Github. Reports a bug, told its not a bug and definitely did not happen.
Like I said, devs on the portmaster discord walked me through troubleshooting to try and solve the issue, and in the end I happened to find that turning off the "Enable SPN" option under SPN Module was the only thing that restored streaming capability. Literally nothing else changed other than that toggle and streaming worked again. If I went back in and turned "Enable SPN" on again, even with SPN Module off and as a free user, streaming stopped working again. It was and still is an easily repeatable issue.

If this is how you approach bugs then it would explain why I and many others still have issues where our internet completely cuts out when we disable or uninstall portmaster. Thats the only reason I'm still using the app, because removing it bricks my PC's internet and I don't want to re-install windows. Again, walked through several hours of troubleshooting in discord with devs, including sending logs several times without a solution. When portmaster is off or uninstalled I get errors on webpages saying the DNS can't be resolved, and one of the devs on discord theorized it might be portmaster's DNS redirection service glitching out and not disabling/uninstalling properly, but I guess that must be in my head too and not actually happening.
